Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM alpine:3.4
-RUN echo "@edge http://nl.alpinelinux.org/alpine/edge/main">>/etc/apk/repositories && \
-    apk update && \
-    apk add ansible@edge curl openssh-client bash git
+FROM alpine:3.8
+RUN apk update && \
+    apk add ansible curl openssh-client bash git
```

Comment:
nit: could use the `--no-cache`, remove the `apk update` and not leave anything in that layer,